lib/reader/src/parser.rs

```diff
@@ -1653,4 +1649,39 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn instruction_encoding() {
+        use cretonne::ir::Inst;
+        use cretonne::entity_map::EntityRef;
+
+        let test_file = parse_test("isa riscv supports_m
```

Comment:
So would all I need to do be drop that IR into a file filetests/parser/instruction_encoding.rs?

Is squashing commits into one a GitHub feature? Still pretty new to pull requests